samples/cpp/lsd_lines.cpp

```diff
@@ -1,11 +1,6 @@
 #include <iostream>
-#include <string>
-
-#include "opencv2/core/core.hpp"
-#include "opencv2/core/utility.hpp"
-#include "opencv2/imgproc/imgproc.hpp"
-#include "opencv2/imgcodecs.hpp"
```

Comment:
Is there a reason why you are removing imgcodecs? Imgproc.hpp is not implicitly including it.